openml-h2o/src/test/java/com/feedzai/openml/h2o/H2OModelProviderTrainTest.java

```diff
@@ -197,35 +198,48 @@ public final void testIsolationForestWithDatasetWithoutTargetVariable() throws M
 
         final MockInstance dummyInstance = new MockInstance(dataset.getSchema(), random);
         final double[] classDistribution = model.getClassDistribution(dummyInstance);
-        assertThat(classDistribution)
-                .as("Scoring instance '%s' succeeds", dummyInstance)
-                .hasSize(2)
-                .matches(predictions -> DoubleStream.of(predictions).sum() == 1.0);
+        assertThat(classDistribution).as("Scoring instance '%s' succeeds", dummyInstance)
+                                     .hasSize(2)
+                                     .matches(predictions -> DoubleStream.of(predictions).sum() == 1.0);
 
         final int classIndex = model.classify(dummyInstance);
-        assertThat(classDistribution[classIndex])
-                .as("The classify method returns the index of the greatest score in the class distribution")
-                .isGreaterThanOrEqualTo(classDistribution[1 - classIndex]);
+        assertThat(classDistribution[classIndex]).as(
+                "The classify method returns the index of the greatest score in the class distribution")
+                                                 .isGreaterThanOrEqualTo(classDistribution[1 - classIndex]);
     }
 
     /**
-     * Tests the handling of a bug in H2O where a serialized Isolation Forest model trained without out-of-bag instances fails
-     * its deserialization due to a {@link NumberFormatException}.
+     * Tests the handling of a bug in H2O where a serialized Isolation Forest model trained without out-of-bag instances
+     * fails its deserialization due to a {@link NumberFormatException}.
      */
     @Test
     public final void testIsolationForestWithNotEnoughInstances() {
         final H2OModelCreator loader = getMachineLearningModelLoader(H2OAlgorithm.ISOLATION_FOREST);
         final Map<String, String> params = H2OAlgorithmTestParams.getIsolationForest();
-        final int sampleSize = Optional.ofNullable(params.get("sample_size"))
-                .map(Integer::parseInt)
-                .orElse(256);
+        final int sampleSize = Optional.ofNullable(params.get("sample_size")).map(Integer::parseInt).orElse(256);
 
         final Random random = new Random(234);
         final Dataset dataset = createDataset(SCHEMA_NO_TARGET_VARIABLE, sampleSize / 2);
 
-        assertThatCode(() -> loader.fit(dataset, random, params))
-                .as("The training of a model with no out of bag instances")
-                .doesNotThrowAnyException();
+        assertThatCode(() -> loader.fit(dataset, random, params)).as(
+                "The training of a model with no out of bag instances").doesNotThrowAnyException();
     }
 
+    /**
+     * Tests that the right exception is thrown when the dataset is empty
+     *
+     * @since 1.0.9
```

Comment:
it's a test, I would argue there's little value in adding this since. Furthermore, if you want to cherry-pick into other hotfix branches you'd need to change it :)

Reply:
As discussed offline with Miguel, since we don't have any automatic way of maintaining versions, I think we have 2 options:
1) We update the since's manually in each backport.
2) We go for the highest version between all the ports (e.g. if porting from/to 1.0.X and 0.5.X) since goes for 1.0.X. My argument is that the latest version is always the most important, so if I have to pick one I'd pick the last one.

Still, regarding having since's in tests, I agree that given the current state of affairs, it's not that important.